suse_migration_services/units/prepare.py

```diff
@@ -81,21 +82,24 @@ def main():
         shutil.copy(
             hosts_setup, '/etc/hosts'
         )
-    if os.path.exists(trust_anchors):
-        certificates = os.listdir(trust_anchors)
-        if certificates:
-            for cert in certificates:
-                log.info(
-                    'Importing certificate: {0}'.format(cert)
-                )
-                shutil.copy(
-                    os.sep.join([trust_anchors, cert]),
-                    '/usr/share/pki/trust/anchors/'
+    for trust_anchor in trust_anchors:
+        root_trust_anchor = os.path.normpath(
+            os.sep.join([root_path, trust_anchor])
+        )
+        if os.path.isdir(root_trust_anchor):
+            certificates = os.listdir(root_trust_anchor)
+            if certificates:
+                Path.create(trust_anchor)
+                for cert in certificates:
```

Comment:
thoughts to use `copytree` instead of `copy` (and the for loop)
```python
log.info(f'Importing certificates: {", ".join(certificates)}')
shutil.copytree(
    root_trust_anchor,
    trust_anchor,
    dirs_exist_ok=True
)
```